src/app/core/services/workWithDate.service.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+export default function (app) {
+  app
+    .service('workWithDate', function () {
+      'ngInject';
+
+      this.transformData = listOfData => {
```

Comment:
it is not correct name for this function

Reply:
renamed it to formatDate